src/transformers/models/bert/tokenization_bert_tf.py

```diff
@@ -114,6 +114,11 @@ def from_tokenizer(cls, tokenizer: "PreTrainedTokenizerBase", **kwargs):  # noqa
         tf_tokenizer = TFBertTokenizer.from_tokenizer(tokenizer)
         ```
         """
+        kwargs.pop("do_lower_case", None)
+        kwargs.pop("cls_token_id", None)
+        kwargs.pop("sep_token_id", None)
+        kwargs.pop("pad_token_id", None)
```

Comment:
I think we can do something like: 
```suggestion
        do_lower_case = kwargs.pop("do_lower_case", None)
        do_lower_case = tokenizer.do_lower_case if do_lower_case is None else do_lower_case
```
for these kwargs. Removing them completely is prone to errors, and people using them but not getting what they want. WDYT 😉

Reply:
Yeah, I agree with your suggestions. I was considering the `from_pretained()` path but this solution is correct also when `from_tokenizer()` is called directly by the user.